src/components/comment/index.jsx

```diff
@@ -23,17 +26,17 @@ const Comment = ({
 
       <span className="controls">
         { !!currentUserIsOwner &&
-          <button id={commentId} onClick={onUserCommentEdit}>
+          <SecondaryButton id={commentId} onClick={onUserCommentEdit}>
             Edit
-          </button>
+          </SecondaryButton>
         }
         { !!currentUserIsOwner &&
-        <button
+        <HyperLink
```

Comment:
When hovering over the Remove button it goes bold and subtly moves the Edit button next to it.